client/src/components/SessionPage.js

```diff
@@ -57,14 +58,15 @@ const getDateAndTime = id => {
 };
 
 const dates = [
-  "October 15, 2017", // TODO change dates
-  "October 16, 2017"
+  "October 8, 2018", // TODO change dates
```

Comment:
what? where did these hid? were they in use?...

Anyway I'd remove the `// TODO change dates` now

Reply:
Note to self: We need to find all the configs in the code and put them in one place.... (not now)